common/errors.h

```diff
@@ -45,17 +50,38 @@ void ProffieOSErrors::font_directory_not_found() {
   beeper.Beep(0.5, 174.61 * 2); // F3
   beeper.Beep(0.5, 146.83 * 2); // D3
   beeper.Beep(0.5, 130.81 * 2); // C3
+  delay(2530);
+#endif
+#endif
+}
+
+void ProffieOSErrors::voice_pack_not_found() {
+  SaberBase::DoEffect(EFFECT_VOICE_PACK_NOT_FOUND, 0);
+  PVLOG_ERROR << "** ERROR - Voice pack not found.\n";
+#ifdef ENABLE_AUDIO
+#ifndef DISABLE_TALKIE
+  talkie.Say(talkie_voice_pack_15, 25);
+  talkie.Say(talkie_not_found_15, 15);
+  delay(2000);
+#else
+  beeper.Beep(1.0, 220.00 * 2); // A4 - Voice
```

Comment:
It would be nice to listen to how all of these things sound.

Reply:
Error in voice pack version and beeps
Voice pack not found and beeps
[vp_errors.wav](https://github.com/user-attachments/files/22194485/vp_errors.wav)